src/main/java/games/strategy/engine/framework/startup/ui/PbemSetupPanel.java

```diff
@@ -425,7 +425,6 @@ public void propertyChange(final PropertyChangeEvent evt) {
       Runtime.getRuntime().addShutdownHook(new Thread(this::writeToDisk));
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
Removing this annotation now flags the following warning for me on line 444: "Type safety: Unchecked cast from Object to HashMap<String,IBean>"

Reply:
can restore, I think save actions was not configured correctly at the time.